client/src/store/graphql/files/models.tsx

```diff
@@ -0,0 +1,60 @@
+import intl from 'react-intl-universal';
```

Comment:
intl can only be used in react components. What you did with the intKey is excellent. Do not use intl here. Plus, if it is a UI column than do not put it in this file. It must be in a react component.